build/python/analyze.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+import importlib
+from importlib.metadata import files
+import sys
+import subprocess
+import traceback
+
+PY_EXTENSION = '.py'
+
+
+def pip_install(package):
+    """Pip install."""
+    subprocess.check_call((sys.executable, '-m', 'pip', 'install', package))
+
+
+def path_to_import(path):
+    """Convert a path to import."""
+    if path.name == '__init__.py':
+        import_path = str(path.parent)
+    else:
+        import_path = str(path)[:-len(PY_EXTENSION)]
+
+    return import_path.replace('/', '.')
+
+
+def analyze(package):
+    """Analyze package."""
+    for path in files(package):
+        # TODO: pyc, C extensions?
+        if path.suffix != PY_EXTENSION:
+            continue
+
+        import_path = path_to_import(path)
+        print('Importing', import_path)
+        try:
+            importlib.import_module(import_path)
+        except:
+            print('Failed to import', import_path)
+            traceback.print_exc()
+
+
+def main():
+    package_with_version = sys.argv[1]
```

Comment:
maybe check length f sys.args and raise ValueError() if not equal to 1?

Reply:
sure, added.